cmd/livepeer/livepeer.go

```diff
@@ -154,6 +155,10 @@ func main() {
 		n.NodeType = core.Broadcaster
 	}
 
+	// if *orchAddr != "" {
+	// 	offchainOrch := n.OrchestratorSelector.NewOffchainOrchestrator(*orchAddr)
```

Comment:
With the way things are now, we'd have to use offchainOrch as an input to `StartMediaServer`, then `gotRTMPStreamHandler`, and `startBroadcast` ... unless we added an address field to `NewLivepeerNode` in `livepeernode.go`